README.md

```diff
@@ -75,11 +77,34 @@ Expected output:
 - metadata files describing each workflow step (yml format)
 - log files (stdout and stderr; ascii format)
 
+#### Workflow with database access
+
+To run workflows which require access to the model database, use the following commands:
+
+```bash
+cwltool --custom-net bridge \
+    --preserve-environment DB_API_PORT \
```

Comment:
Not a big deal, but I hope this can simplified in the future (not now).

Reply:
Yes, I hope so too. I spent quite some time trying to figure out how to pass on `.env` to the docker but failed.